test/es6-flow-component.jsx

```diff
@@ -0,0 +1,33 @@
+const React = require('react');
+
+// Contrived example to show how one might use Flow type annotations
+function countTo(n:number):string {
+  var a = [];
+  for (var i = 0; i < n; i++ ) {
+    a.push(i + 1);
+  }
+  return a.join(', ');
+}
+
+class Index = extends React.Component {
+  static propTypes() {
+    return {
+      title: React.PropTypes.string
+    };
+  }
```

Comment:
Let's actually specify `propTypes` at the end of the class (I actually think what you have here won't properly check props, I didn't think we'll ever call a method, only look at properties on `Class.propTypes`).

I think babel supports static property initializers but let's just do it after the class [like we've specified in the React docs](http://facebook.github.io/react/docs/reusable-components.html#es6-classes).

```js
Class.propTypes = {
  title: React.PropTypes.string
};
```